qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -113,6 +113,9 @@ def main(self):
         self.add_options(parser)
         (self.options, self.args) = parser.parse_args()
 
+        if os.getenv("QT", ""):
+            self.options.noshutdown = True
```

Comment:
The reason why I added the noshutdown=True is, because i think running a RPC test with the UI needs post-run user based verification (does the GUI looks like expected?). As long as there is no automatic screenshot creating (and maybe auto-comparing) I think the noshutdown flag is appropriate.
The `QT=1` env flag is a helper IMO. It's not really necessary because one could also pass the path to bitcoin-qt in the `BITCOIND` env var.